Documentation/HowToGuides/CreatingProxyActionsAndChainingActions/README.md

```diff
@@ -0,0 +1,82 @@
+# Creating Proxy Actions And Chaining Actions
+
+> * Level: Beginner
+>
+> * Reading Time: 5 minutes
+>
+> * Checked with: Unity 2018.3.14f1
+
+## Introduction
+
+It is sometimes desirable to have many different input actions all have the same outcome. It is possible to simply do this by wiring up all of the events in all of the actions to have the same output, however this can get cumbersome and hard to maintain.
+
+The Zinnia Actions can be used as proxies to enable chaining of Actions to all drive the same outcome defined in one place.
+
+Following on from the scene created in Adding A Unity Button Action, we’re going to move the Action functionality to a Proxy Action and then chain the button press Action and keyboard Action to this newly created Proxy Action. This will move our output functionality into one place and allow us to call it from many different actions.
+
+### Prerequistes 
+
+* A Unity Button Action exists in the scene. See [Using The Unity Button Action].
+
+## Let's Start
+
+### Step 1
+
+Create an empty `GameObject` in the Unity Hierarchy and rename it to `FloorToggler` then click the `Add Component` button and select the `Boolean Action` component.
+This newly created `Boolean Action` will be the action that defines how we toggle the active state of the `Floor` GameObject.
+
+![Creating The Floor Toggler](assets/images/CreatingTheFloorToggler.png)
+
+### Step 2
+
+Click the `+` symbol in the bottom right corner of the `Activated` event parameter in the `Boolean Action` component and then drag and drop the `Floor` GameObject into the box that appears and displays `None` `(Object)`.
+
+![Creating The Boolean](assets/images/CreatingTheBoolean.png)
+
+### Step 3
+
+Do the same as Step 2 but for the `Deactivated` event parameter in the `Boolean Action` component, but tick the checkbox under the `GameObject.SetActive function`.
```

Comment:
don't include function in the backtick

`GameObject.SetActive function` 

should be

`GameObject.SetActive` function